src/lib/VictoryConditon.ts

```diff
@@ -0,0 +1,37 @@
+import { SimulatorState } from './SimulatorState';
+
+export interface VictoryCondition {
+    name: string;
+    description: string;
+    isMet(simulatorState: SimulatorState): boolean;
+}
+
+export const TimeVictory: VictoryCondition = {
+    name: 'Time victory',
+    description: 'You managed to survice one year!',
+    isMet: (simulatorState: SimulatorState) => simulatorState.currentState.days >= 365
```

Comment:
The condition that is evaluated by the simulator is defined as a function. In this case we can trivially implement a condition that finishes the game after 365 game days have elapsed